src/components/Subpage.tsx

```diff
@@ -0,0 +1,128 @@
+import React from "react";
+import { Card, CardProps } from "antd";
+import { GatsbyImage, getImage } from "gatsby-plugin-image";
+import { Diagram } from "../types";
+
+const {
+    container,
+    card,
+    dataTable,
+    dataRow,
+    dataLabel,
+    diagramCard,
+    diagramCardHeader,
+    diagramCardFooter,
+    diagramImage,
+    legendText,
+} = require("../style/disease-subpage.module.css");
+
+export interface DataRow {
+    label: string;
+    value: string;
+    formatClass?: string;
+}
+
+export interface SubpageProps {
+    data: {
+        rows: DataRow[];
+        diagrams?: Diagram[];
+        legendContent?: React.ReactNode;
+    };
+    headerLeadText?: string;
+}
+
+interface DiagramCardProps extends CardProps {
+    title?: string;
+    footer?: React.ReactNode;
+}
+
+export const DiagramCard: React.FC<DiagramCardProps> = ({
+    children,
+    title,
+    footer,
+    className,
+}) => (
+    <Card
+        className={className}
+        title={title && <div>{title}</div>}
+    >
+        {children}
+        {footer && <div className={diagramCardFooter}>{footer}</div>}
+    </Card>
+);
+
+const formatTextWithGeneLocations = (text: string, className: string) => {
```

Comment:
this is very specific to editing design, and should be in there (or in an associated util file)  and not in a shared component because I don't think it will be used anywhere else